embeddings/src/embeddings/generateEmbeddings.ts

```diff
@@ -27,8 +26,9 @@ type ChunkedFiles = {
  */
 const CHUNK_SIZE = 300;
 
-const SHARED_DIR = './src/shared/docs';
-const files = fs.readdirSync(SHARED_DIR);
+// Relative to the execution of the server in .waps/out/server
+const DOCS_DIR = '../../../src/docs';
```

Comment:
Oh this is weird, right? Is it? Can't this be solved in some nicer way?

So docs/ -> these are really static files that we want to ship with the server, and have server load them while executing? Why not put them inside of `server/` then, that should move them to .wasp/out/server then and they should keep the same relative position, I think?

This might be related to https://github.com/wasp-lang/wasp/issues/1564 .

I get that this is old thing that got "worsened" with the restructuring, so I don't think this should stop the review, but still wanted to discuss it regardless of this PR's lifecycle.

Reply:
>  Why not put them inside of server/ then, that should move them to .wasp/out/server then and they should keep the same relative position, I think?

Could you explain how would you do this? I'm not sure if we have a mechanism for copying over static files that got included in the Node.js world